designcompose/src/main/java/com/android/designcompose/DocContent.kt

```diff
@@ -24,6 +24,7 @@ import com.android.designcompose.common.FeedbackImpl
 import com.android.designcompose.common.GenericDocContent
 import com.android.designcompose.common.decodeDiskBaseDoc
 import com.android.designcompose.common.decodeServerBaseDoc
+import com.google.protobuf.kotlin.get
```

Comment:
nit: is this unused import?

Reply:
Doesn't look like it. Maybe because it's an extension function